barretenberg/cpp/src/barretenberg/vm2/simulation/lib/raw_data_dbs.hpp

```diff
@@ -37,6 +39,7 @@ class HintedRawContractDB final : public ContractDBInterface {
     unordered_flat_map<AztecAddress, ContractInstanceHint> contract_instances;
     unordered_flat_map<ContractClassId, ContractClassHint> contract_classes;
     unordered_flat_map<ContractClassId, FF> bytecode_commitments;
+    unordered_flat_map<std::pair<AztecAddress, FunctionSelector>, std::string> debug_function_names;
```

Comment:
In my upcoming PR I will probably make some struct like `DebugFunctionNameKey { address, selector }` just FYI but this is all good for now!